src/test/java/PokeLifeBotByMetin/tests/AAllTests.java

```diff
@@ -0,0 +1,58 @@
+package PokeLifeBotByMetin.tests;
+
+import PokeLifeBotByMetin.pages.*;
+import org.testng.annotations.Test;
+
+public class AAllTests extends BaseTest {
```

Comment:
AAll -> All :)

Reply:
I didn't found a better option to make class 1st on the list XD